--- 8503_postprocess_freq_analysis_bro_http.conf	2018-12-20 14:37:45.158742891 +0100
+++ 8503_postprocess_freq_analysis_bro_http.conf.orig	2018-12-20 14:34:22.069580243 +0100
@@ -14,7 +14,7 @@
       }
       rest {
         request => {
-          url => "http://freqserver:10004/measure1/%{freq_virtual_host}"
+          url => "http://localhost:10004/measure/%{freq_virtual_host}"
         }
         sprintf => true
         json => false
