--- 8505_postprocess_freq_analysis_bro_x509.conf	2018-12-20 14:38:02.894967237 +0100
+++ 8505_postprocess_freq_analysis_bro_x509.conf.orig	2018-12-20 14:34:22.073580257 +0100
@@ -21,7 +21,7 @@
       }
       rest {
         request => {
-          url => "http://freqserver:10004/measure1/%{freq_common_name}"
+          url => "http://localhost:10004/measure/%{freq_common_name}"
         }
         sprintf => true
         json => false
@@ -46,7 +46,7 @@
       }
       rest {
         request => {
-          url => "http://freqserver:10004/measure1/%{freq_common_name}"
+          url => "http://localhost:10004/measure/%{freq_common_name}"
         }
         sprintf => true
         json => false
@@ -71,7 +71,7 @@
       }
       rest {
         request => {
-          url => "http://freqserver:10004/measure1/%{freq_common_name}"
+          url => "http://localhost:10004/measure/%{freq_common_name}"
         }
         sprintf => true
         json => false
